### ISSUE TITLE
github actions: automatically check R packages

### ISSUE THREAD

**Issue by SamCH93** [SamCH93/ReplicationSuccess#56]
This could be a good alternative to travis and  I think it is free for *public* repositories:
"GitHub Actions is available with GitHub Free, GitHub Pro, GitHub Free for organizations, GitHub Team, GitHub Enterprise Cloud, GitHub Enterprise Server, and GitHub AE. GitHub Actions is not available for private repositories owned by accounts using legacy per-repository plans. For more information, see "GitHub's products."
(<https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners>)

**Comment by florafauna** [comment]
yes, indeed. the repository is now public. But maybe it's better to create an experimental (new) repository and try it there.
